pkg/types/report.go

```diff
@@ -50,11 +50,12 @@ const (
 	ClassLicenseFile = "license-file" // For detected licenses in files
 	ClassCustom      = "custom"
 
-	// ComplianceNsa is the compliance checks for nsa
-	ComplianceK8sNsa   = Compliance("k8s-nsa")
-	ComplianceK8sCIS   = Compliance("k8s-cis")
-	ComplianceAWSCIS12 = Compliance("aws-cis-1.2")
-	ComplianceAWSCIS14 = Compliance("aws-cis-1.4")
+	ComplianceK8sNsa             = Compliance("k8s-nsa")
+	ComplianceK8sCIS             = Compliance("k8s-cis")
+	ComplianceK8sPSSBaseline     = Compliance("k8s-pss-baseline")
+	ComplianceK8sPSSRestricted   = Compliance("k8s-pss-restricted")
```

Comment:
need also to add it to [compliances array ](https://github.com/aquasecurity/trivy/blob/main/pkg/types/report.go#L11)

Reply:
thanks. fixed now